### ISSUE TITLE
The coupled equations are not convergent

### ISSUE THREAD

**Issue by SSY0218** [lululxvi/deepxde#197]
Hi, LULU. Thank you for sharing the package, which I am currently using to implement a Thermoelastic coupling equation. There are three inputs and three outputs of the PDE equations. At present, only one output can be restricted to the boundary condition , the rest of the outputs can`t converge.
I have checked the other issuse, you have mentioned loss weight,  increase the depth and width of the network and num_ domain，num_ boundary. But it still can't converge. I 'd like to ask you **how to make three equations converge at the same time.** My boundary conditons and experimental results are as follows，
![myplot](https://user-images.githubusercontent.com/76893113/103481545-ab975200-4e16-11eb-87ef-c2cf19052439.png)
![myplot2](https://user-images.githubusercontent.com/76893113/103481546-acc87f00-4e16-11eb-95dd-6af61ddcaa74.png)
![myplot3](https://user-images.githubusercontent.com/76893113/103481547-adf9ac00-4e16-11eb-87c5-b368ec9027b3.png)
<img width="831" alt="QQ20210103-152107@2x" src="https://user-images.githubusercontent.com/76893113/103481548-ae924280-4e16-11eb-9f76-bd86cdbff742.png">
The first graph is the curve of T, the second graph is the curve of ur, and the third graph is the curve of UZ. The distribution of T satisfies the boundary condition and PDE equation，but Ur and UZ do not. 
The followings are my setting of the model: 
r=15
z=30
t=10
layer= [3 , [60]*5, 3 ] 
num_domain=8000,
num_boundary=8000, 
num_initial=8000,
epochs=45000,loss_weights=[1,1e-20,1e-20,1,1,1,1,1,1,1,1,1,1]

**Comment by lululxvi** [comment]
Do you try to re-scale the problem such that T, Ur, and Uz are all of order 1? Also, you can try to impose the BC for Uz and Ur exactly by modifying the network using `net.apply_output_transform` (check other issues for this).

**Comment by SSY0218** [comment]
I have checked issue 36, adding hard boundary condition to force converge. i`m wondering whether changing the output of neural network will affect the convergence of other places. For example,  in issue 36 , you have mention using net.outputs_modify(lambda x, y: x + x * (1 - x) * y) which can easily get y(0) = 0, y(1) = 1, but when x =2 ,the original output is y and now is 2y+2. as you can see, the other output of neural network will be enlarged by different times. will this effect the final result?

**Comment by lululxvi** [comment]
No. You will always train a network. It would not be a problem.

**Comment by SanjuSoni** [comment]
@SSY0218 
have you got your solution?
